src/net_processing.cpp

```diff
@@ -2495,6 +2495,7 @@ uint32_t PeerManagerImpl::GetFetchFlags(const Peer& peer) const
 void PeerManagerImpl::SendBlockTransactions(CNode& pfrom, Peer& peer, const CBlock& block, const BlockTransactionsRequest& req)
 {
     BlockTransactions resp(req);
+    unsigned int tx_missing_size = 0;
```

Comment:
Is this variable being used ?

Reply:
Thanks for catching this, it's printed below in the added `LogDebug()` message but I forgot to increment this.